File: kala-game-service/src/main/kotlin/org/jesperancinha/games/kalagameservice/service/PlayerService.java
package org.jesperancinha.games.kalagameservice.service;

import org.jesperancinha.games.kalagameservice.model.Player;

public interface PlayerService {
    Player createPlayer(String username);

    Player createOrFindPlayerByName(String username);

    void leaveCurrentGame(String name);
}

File: kala-game-service/src/main/kotlin/org/jesperancinha/games/kalagameservice/service/PlayerService.kt
package org.jesperancinha.games.kalagameservice.service

import org.jesperancinha.games.kalagameservice.model.Player

interface PlayerService {
    fun createPlayer(username: String?): Player?
    fun createOrFindPlayerByName(username: String): Player?
    fun leaveCurrentGame(name: String?)
}